mkosi

```diff
@@ -765,7 +765,7 @@ def create_image(args: CommandLineArguments, root: str, for_cache: bool) -> Opti
         table, run_sfdisk = determine_partition_table(args)
 
         if run_sfdisk:
-            run(["sfdisk", "--color=never", f.name], input=table.encode("utf-8"), check=True)
+            run(["sfdisk", "--color=never", f.name], input=table.encode("utf-8"))
             run(["sync"])
```

Comment:
This one wasn't checked but maybe we should check the result of sync? I don't think we expect it to fail.

Reply:
I think it's fine if the sync is checked, if it fails we shouldn't go ahead anyway.